deps/v8/build/toolchain.gypi

```diff
@@ -995,7 +995,7 @@
           'VCCLCompilerTool': {
             'Optimization': '0',
             'conditions': [
-              ['component=="shared_library"', {
+              ['component=="shared_library" or node_shared=="true"', {
```

Comment:
In v6.x this looks like: 

 ['component=="shared_library" or force_dynamic_crt==1', {

Further, in master I think it still uses force_dynamic_crt

I would have expected the backport for 4.x to be consistent ? 

So this should be consistent with what we landed in v8 right ?